portshaker.subr.in

```diff
@@ -520,34 +506,19 @@ run_portshaker_command()
 			esac
 			;;
 		merge_to)
-			_args=`getopt ab:m:pt: ${_portshaker_extra_args}`
-			if [ $? -ne 0 ]; then
-				portshaker_usage
-			fi
-			set -- ${_args}
 			_auto_install=0
 			_master=""
 			_target=""
 			_tinderbox_builds=""
-			for _i; do
+			set -- "$@"
```

Comment:
We can drop this line.

Reply:
Yes you're right. That's superfluous. `getopts` uses the current script or function arguments anyway.

Reply:
Fine, I removed them in c6d0733d672dd71fd18561b4f2230718b28204d1.